interface/skin.dmf

```diff
@@ -1423,7 +1423,6 @@ window "infowindow"
 		is-default = true
 		saved-params = ""
 		highlight-color = #00aa00
-		allow-html = true

```

Comment:
Does this break the colored text for the different types of examines?

Reply:
Oddly enough, no.   They were colored before this was added a few days ago, and still work without it.